test/discovery_and_monitoring/rs/compatible.json

```diff
@@ -16,7 +16,7 @@
               "b:27017"
             ],
             "minWireVersion": 0,
-            "maxWireVersion": 21
+            "maxWireVersion": 6
```

Comment:
Guessing this is just a spec requirement change?

Reply:
Yes.